.circleci/config.yml

```diff
@@ -77,8 +77,8 @@ references:
   restore-yarn-cache: &restore-yarn-cache
     name: 'Restore yarn cache'
     keys:
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v2-yarn-cache-{{ checksum ".yarnrc.yml" }}
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v2-yarn-cache
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v3-yarn-cache-{{ arch }}-{{ checksum ".yarnrc.yml" }}
```

Comment:
Need to add `{{ arch }}` so linux cache is not used for `wp-desktop-mac`

Reply:
Nice catch!